### PR TITLE
DOCS-1824: Adds why some payment methods are not visible in checkout for Magento 2

### PR DESCRIPTION
**Pull request by Atiq-MultiSafepay**

### CODE REVIEW

**Review by Vinod-MultiSafepay — CHANGES_REQUESTED** [review]
What about:

- The payment method has been enabled in the wrong store view

**Review by Kris-MultiSafepay — COMMENTED** [review]
Please revise

File: content/integrations/plugins/magento2/old/faq/why-is-a-specific-payment-method-not-visible-in-my-checkout.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ layout: "faqdetail"
 read_more: "."
 ---
 
-When you enabled a payment method, it should always be visible - even when your API key is incorrect -.
+When you enabled a payment method, it should always be visible, even when your API key is incorrect.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/old/faq/why-is-a-specific-payment-method-not-visible-in-my-checkout.md:9]: When you enabled a payment method, it should always be visible, even when your API key is incorrect.
```suggestion
When you enabled a payment method, it should always be visible, even when your API Key is incorrect.
```

File: content/integrations/plugins/magento2/old/faq/why-is-a-specific-payment-method-not-visible-in-my-checkout.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ layout: "faqdetail"
...
 
-Be sure that when you use the [Rico Neitzel Payment Filter](https://github.com/riconeitzel/PaymentFilter) that you set the payment method as active in that plugin.
+Payment methods may also not be visible in the checkout due to the following:
```
[the remark this answers is not in the document]
Comment (Atiq-MultiSafepay) [reply content/integrations/plugins/magento2/old/faq/why-is-a-specific-payment-method-not-visible-in-my-checkout.md:11]: I will create an individual ticket to change this. Currently, i get a merge conflict in visual processing a change. All other suggested commits have been pushed.

File: content/integrations/plugins/magento2/old/faq/why-is-a-specific-payment-method-not-visible-in-my-checkout.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ layout: "faqdetail"
 read_more: "."
 ---
 
-When you enabled a payment method, it should always be visible - even when your API key is incorrect -.
+When you enabled a payment method, it should always be visible, even when your API Key is incorrect.
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/old/faq/why-is-a-specific-payment-method-not-visible-in-my-checkout.md:9]:
```suggestion
When you enabled a payment method, it should always be visible, even when your API key is incorrect.
```

**Review by Kris-MultiSafepay — APPROVED** [review]

**Pull request merged**